### ISSUE TITLE
[Feature] cheatsheet improvements

### ISSUE THREAD

**Issue by l4v3nx** [end-4/dots-hyprland#607]
**What would you like to be added?**

is there any way to add combined keybinds (two keybinds doing one action) like on screenshot?
![image](https://github.com/end-4/dots-hyprland/assets/58698398/49d51b82-711f-48cc-8ede-a46a4ce2a76c)

also it will be cool is you add general keybind for similar action (like in old cheatsheet)
![image](https://github.com/end-4/dots-hyprland/assets/58698398/79d98c9f-d717-4b82-ba3b-7465fd76237d)
![image](https://github.com/end-4/dots-hyprland/assets/58698398/40746aec-80b4-4397-b3aa-2f9707a4786d)
![image](https://github.com/end-4/dots-hyprland/assets/58698398/bb847e13-0277-4096-995d-d3b03c8d68e2)

**How will it help**

i'm trying to recreate the old cheatsheet and because there aren't general keybinds and in my cheatsheet text goes under screen (i don't want to use tabs, it's inconvenient to use keybind on my keyboard and with broken arm)
![image](https://github.com/end-4/dots-hyprland/assets/58698398/7eab4b2c-1d52-44db-b46d-30568a0256fb)

**Comment by l4v3nx** [comment]
this cheatsheet https://github.com/l4v3nx/dots-hyprland/blob/ca624baf14f51e9d97b97e78cbc8c8ea2e52f3c4/.config/hypr/custom/keybinds.conf

**Comment by end-4** [comment]
yea i'll have to improve the config parsing script

**Comment by end-4** [comment]
![image](https://github.com/end-4/dots-hyprland/assets/97237370/bf834623-baa1-4d7c-aef3-94ba752875bf)

**Comment by end-4** [comment]
i haven't done the OR yet but it'c already nicer
hmm

**Comment by l4v3nx** [comment]
yeah, it pretty good. I've almost done configuring cheatsheet, tnx))))

**Comment by l4v3nx** [comment]
![image](https://github.com/end-4/dots-hyprland/assets/58698398/7afd93c7-e9ff-42f7-9590-059b3636043f)

looks clean ngl
